README.md

```diff
@@ -6,8 +6,8 @@ UNOFFICIAL FFT interfaces for Kokkos C++ Performance Portability Programming Eco
 
 KokkosFFT implements local interfaces Kokkos and de facto standard FFT libraries, including fftw, cufft and hipfft. 
 "Local" means not using MPI, or running within a
-single MPI process without knowing about MPI. We are inclined to implement the numpy FFT interfaces based on Kokkos.
-Here is the example for 1D case in python and KokkosFFT.
+single MPI process without knowing about MPI. We are inclined to implement the [numpy.fft](https://numpy.org/doc/stable/reference/routines.fft.html) interfaces based on [Kokkos](https://github.com/kokkos/kokkos). 
```

Comment:
I do not think that we implement the numpy interface. numpy-like interface (adapted for Kokkos) seems more accurate.